bin/console

```diff
@@ -14,7 +14,7 @@ set_time_limit(0);
 /**
  * @var Composer\Autoload\ClassLoader $loader
  */
-$loader = require __DIR__.'/autoload.php';
+$loader = require __DIR__.'/../app/autoload.php';
 
 $input = new ArgvInput();
 $env = $input->getParameterOption(array('--env', '-e'), getenv('SYMFONY_ENV') ?: 'dev');
```

Comment:
I agree ... but let's do that in a separate PR to not complicate this one too much. In fact, let's push PHP to use as many PHP 5.5 features as we can, not only the short array syntax.